lib/ts/recipe/session/types.ts

```diff
@@ -29,26 +28,8 @@ export type RecipeEvent =
 
 export type RecipeEventWithSessionContext = RecipeEvent & { sessionContext: SessionContextType };
 
-export type InputType = {
-    apiDomain?: string;
-    apiBasePath?: string;
-    sessionScope?: string;
-    sessionExpiredStatusCode?: number;
-    autoAddCredentials?: boolean;
-    isInIframe?: boolean;
-    cookieDomain?: string;
-    preAPIHook?: (context: {
-        action: "SIGN_OUT" | "REFRESH_SESSION";
-        requestInit: RequestInit;
-        url: string;
-    }) => Promise<{ url: string; requestInit: RequestInit }>;
-    onHandleEvent?: (event: RecipeEvent) => void;
-    override?: {
-        functions?: (
-            originalImplementation: RecipeInterface,
-            builder?: OverrideableBuilder<RecipeInterface>
-        ) => RecipeInterface;
-    };
+export type InputType = WebJSInputType & {
+    onHandleEvent?: (event: RecipeEventWithSessionContext) => void;
```

Comment:
Where is the postAPIHook? Am i confusing with something else?

Reply:
`WebJSInputType` contains both pre and post API hooks. The only different in webjs types and this one is the onHandleEvent which needs to session context so ive commented it